src/pages/_app.tsx

```diff
@@ -0,0 +1,11 @@
+import { ChakraProvider } from '@chakra-ui/react'
+
+function MyApp({ Component, pageProps }) {
```

Comment:
Please rename it instead of using `MyApp`.

Reply:
```suggestion
function App({ Component, pageProps }) {
```